### ISSUE TITLE
RunTools Icons not displaying correctly

### ISSUE THREAD

**Issue by david1309** [ipython-contrib/jupyter_contrib_nbextensions#936]
Hello,

I am trying to use the` Runtools nbextension` on my `Jupyter Notebook,` but when I enable it the icons do not display correctly and thus, I can't really use it.

![screenshot from 2017-03-31 20-36-47](https://cloud.githubusercontent.com/assets/4961892/24574014/09c7719e-1652-11e7-96c5-d2444c1b536d.png)

**Comment by juhasch** [comment]
The colors of your Jupyter notebook look different. What notebook version, Browser type, OS, and additional extensions or modifications do you use ?

**Comment by david1309** [comment]
Yes, the colors are different since I changed Jupyter's notebook theme to **monokai**. I did this using[ this extension](https://github.com/dunovank/jupyter-themes) and downloaded the CSS scheme from [here](https://github.com/transcranial/jupyter-themer/tree/master/jupythemer/styles/color).

**Version:** Jupyter 4.3.0
**Browser:** Chrome
**OS:** Ubuntu 16.04 LTS
**Additional Extensions:** Change theme to Monokai using [this](https://github.com/dunovank/jupyter-themes)

**Comment by jfbercher** [comment]
Can you try to disable this theme and try the runtools extension again? It is possible that your two issues, this one and #937, are related to a missing/changed font and that can be linked to the theme you have installed.

**Comment by juhasch** [comment]
These are `fontawesome` icons. You can try them individually, e.g. type
```
%%html
<i class="fa fa-step-forward"></i> fa-step-forward
```
In a codecell. You should an icon like the one shown here: http://fontawesome.io/icons/

**Comment by david1309** [comment]
@jfbercher you are right, when disabling the monokai theme, the icons from the  `Run Tools` extension actually displayed correctly.

![screenshot from 2017-04-07 15-33-46](https://cloud.githubusercontent.com/assets/4961892/24818890/e4a234f2-1ba7-11e7-9183-8552aa4b3112.png)

Is there any way I could get this extension to work with a different theme to the default Jupyter Notebook theme ?

@juhasch , I tried what you mentioned and in both, default theme and monokai theme the icons display correctly.

![screenshot from 2017-04-07 15-36-57](https://cloud.githubusercontent.com/assets/4961892/24818985/57a564d8-1ba8-11e7-8bb9-c43c26494a92.png)

![screenshot from 2017-04-07 15-38-21](https://cloud.githubusercontent.com/assets/4961892/24818992/5d2c01be-1ba8-11e7-8026-24152f98cf29.png)

**Comment by david1309** [comment]
Forgot to mention that reseting Jupyter's default theme did not solve anything regarding #937 @jfbercher

**Comment by jcb91** [comment]
Weird. It looks most like a missing bit of font awesome, but the fact that the main toolbar buttons, and the cell HTML output render ok in both suggests that it's something more specific to the way runtools is attempting to use the fontawesome icons. I note that a number of them seem to be using the older-style `icon-step-forward` class names instead of the newer `fa-step-forward`, which might be relevant. Alternatively, the mixing of the `btn` and `fa` classes on he same element (normally in notebook the `fa` is applied to a child element of the `btn`) may be causing problems?

**Comment by juhasch** [comment]
The icons are combinations of fontawesome-icons and are defined in `main.css`
For example
```
%%html
<i class="fa icon-run-to" />
```
is defined as `.icon-run-to:before{content:"\f04b\f10c"}`

What happens if you try
```
%%html
<div class="btn-group">\
    <button type="button" id="run_c" class="btn btn-primary fa fa-step-forward"></button>
    <button type="button" id="run_ca" class="btn btn-primary fa icon-run-to"></button>
    <button type="button" id="run_cb" class="btn btn-primary fa icon-run-from"></button>
    <button type="button" id="run_a" class="btn btn-primary fa icon-run-all"></button>
    <button type="button" id="run_af" class="btn btn-primary fa icon-run-all-forced"></button>
    <button type="button" id="run_m" class="btn btn-primary fa icon-run-marked"></button>
    <button type="button" id="interrupt_b" class="btn btn-primary fa fa-stop"></button>
</div>
```

**Comment by david1309** [comment]
@juhasch , Here what I get both running it with the _monokai_ themes (using` Jupyter Theme` extension) and with the default jupyter theme:

![screenshot from 2017-04-10 19-25-49](https://cloud.githubusercontent.com/assets/4961892/24887724/b4230788-1e23-11e7-8d9a-0998b4475f70.png)

![screenshot from 2017-04-10 19-25-19](https://cloud.githubusercontent.com/assets/4961892/24887726/b80a6ada-1e23-11e7-998e-56fd1990c60f.png)

**Comment by DimTrigkakis** [comment]
> Alternatively,
> the mixing of the btn and fa classes on he same element (normally in notebook the fa is applied to a
> child element of the btn) may be causing problems?

In my case, that was exactly the issue! After removing btn from the class I could see all buttons instead of the square "not-found" icons.
